<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEdgeWindow</class>
 <widget class="QDialog" name="addEdgeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="mainArea" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255, 0.9);
border-radius: 30px;</string>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font-family: 'MiSans Demibold';
font-size: 25px;</string>
    </property>
    <property name="text">
     <string>添加连边</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgba(195, 195, 195, 0.5);
height: 40px;
width: 150px;
font-family: 'MiSans';</string>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>站点名称1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>站点名称2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>连接线路</string>
    </property>
   </widget>
   <widget class="QComboBox" name="line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans Light';
border-radius:0px;
padding-left:10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="station1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
padding-left:10px;
font-family:'MiSans Light';</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="station2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
padding-left:10px;
font-family:'MiSans Light';</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addLine">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'MiSans'; font-size: 20px;
background-color: rgba(195, 195, 195, 0.5);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
